### PR TITLE
Feature/long term storage ddev testing

### PR DESCRIPTION
**Pull request by finnlewis**
@Adnan-cds just adding specific notes on setting up ddev to test, as it took me a little while to work it out!

These OK in the README or would you prefer them in another file?

### CODE REVIEW

File: modules/localgov_forms_lts/README.md [hunk 1]
```diff
@@ -33,3 +33,75 @@ To inspect Webform submissions kept in Long term storage, look for the "LTS" tab
...
+Edit sites/default/settings.php and add a new database connection string at the
+end of the file.
+
+```
+// Database connection for localgov_forms_lts.
+$databases['localgov_forms_lts']['default'] = [
+  'database'  => 'localgov_forms_lts',
+  'username'  => 'db',
+  'password'  => 'db',
+  'host'      => 'db',
```
Comment (Adnan-cds) [inline modules/localgov_forms_lts/README.md:65]: Perhaps use slightly better placeholders? :)  Something like `USERNAME-GOES-HERE`, `PASSWORD-GOES-HERE`, `HOSTNAME-GOES-HERE`, etc.?

**Review by Adnan-cds — COMMENTED** [review]
I don't know about the correctness of the ddev instructions here.  I will just trust you on this:)

File: modules/localgov_forms_lts/README.md (hunk 1 above)
Comment (finnlewis) [reply modules/localgov_forms_lts/README.md:65]: For ddev, the username, password and host are all actually db! :)

**Comment by Adnan-cds** [comment]
All good then :)

**Pull request merged**